build_product

```diff
@@ -79,7 +79,7 @@ _arg_bash_scripts="on"
 _arg_datastream_only="off"
 _arg_profiling="off"
 _arg_log="off"
-
+_arg_generate_oval_document_per_rule="off"
```

Comment:
I'd prefer that this would be turned on by default and at the same time remove `--build-ovals-dir` from the invocations of `combine_ovals.py`.